src/DbTests/facil.yaml

```diff
@@ -192,6 +192,8 @@ rulesets:
           tvp:
             type: dbo.SingleColNonNull
 
+      - for: ManyColumns.sql
```

Comment:
Empty `for` has no effect, this can be removed.

Reply:
Removed